projectDocs/dev/createDevEnvironment.md

```diff
@@ -53,7 +53,7 @@ To replicate the production build environment, use the 3.11.x minor version of P
 			* Once selected, ensure "C++ Clang tools for Windows" is included under the optional grouping.
 	* On the Individual components tab, ensure the following items are selected:
 		* Windows 11 SDK (10.0.22621.0)
-		* MSVC v143 - VS 2022 C++ ARM64 build tools
+		* MSVC v143 - VS 2022 C++ ARM64/ARM64EC build tools

```

Comment:
@michaelDCurran - is this true?

Reply:
Yes, it looks like ARM64 and ARM64EC build tools are now bundled together (at least according to the installer UI).
So I think this is a correct change.
Though oddly in the `.vsconfig` file they are still always split into two separate components when exporting.